### ISSUE TITLE
resume training

### ISSUE THREAD

**Issue by rcogrann** [JiahuiYu/generative_inpainting#278]
Hello, 

I would like to resume training of your release_places2 model, for training on 16 bits images, is there a mean to freeze some layers or to modify the learning rate for the network to focus on learning dataset-specific features in the subsequent layers ?

**Comment by JiahuiYu** [comment]
You can do that by setting the trainable variables of first several layers to trainable=False. There should be many solutions in the tensorflow community for this purpose. Please have a re-search.
